src/test/resources/Features/verifyApiReturnsCorrectStatusCode.feature

```diff
@@ -0,0 +1,161 @@
+@Regression
```

Comment:
All the scenarios can be combined to a single one but it could be a bit confusing while trying to add more endpoints in near future. That's one of the reasons I have separated the scenarios based on the controllers.